client/src/templates/Challenges/components/Challenge-Description.js

```diff
@@ -9,11 +9,16 @@ const propTypes = {
   section: PropTypes.string
 };
 
+function emptyInstruction(instructions) {
+  return (/<\s*section\s*id\s*=\s*"\s*instructions"\s*>\s*<\s*\/\s*section/)
```

Comment:
need to check both `'|"`

and could you be more accurate:
- check whole string `/^...$/`
- remove redundant `\s*` like in `<\s*section` or `"\s*instruction"`
- `section\s+id`